src/file/file.service.spec.ts

```diff
@@ -154,6 +154,50 @@ describe('File Logic', () => {
     });
   });
 
+  describe('#createUpdate', () => {
+    it('should return a new upload', async () => {
+      const key: string = KEY;
+      const key2: string = KEY;
+      const key3: string = KEY;
+      await FileService.create(bucket, 'file.txt', USER.id, 'text', key, key2, size);
+      const newUpdate: IUpload = await UploadService.createUpdate(key, bucket, 'file.txt',  USER.id, key3, size).should.eventually.exist;
+      expect(newUpdate).to.exist;
+      expect(newUpdate.bucket).to.be.equal(bucket);
+      expect(newUpdate.name).to.be.equal('file.txt');
+      expect(newUpdate.key).to.be.equal(key);
+    });
+
+    it('should return file not exist', async () => {
+      await UploadService.createUpdate(testUpload.key, testUpload.bucket, testUpload.name, USER.id, null, 256)
+      .should.eventually.be.rejectedWith(FileNotFoundError);
+    });
+
+    it('should throw exceeded quota error', async () => {
+      const key: string = KEY;
+      const key2: string = KEY;
+      const key3: string = KEY;
+      await FileService.create(bucket, 'file.txt', USER.id, 'text', key, key2, size);
+      await UploadService.createUpdate(key, bucket, 'file.txt',  USER.id, key3, 101 * 1024 * 1024 * 1024)
+        .should.eventually.be.rejectedWith(QuotaExceededError);
+    });
+
+    it('should increase quota usage by the size of the upload', async () => {
+      const oldQuota = await QuotaService.getByOwnerID(USER.id);
+      const key: string = KEY;
+      const key2: string = KEY;
+      const key3: string = KEY;
+      await FileService.create(bucket, testUpload.name, USER.id, 'text', key, key2);
+      const newUpload: IUpload = await UploadService.createUpdate(key, bucket, testUpload.name,  USER.id, key3, 6 * 1024).should.eventually.exist;
+      expect(newUpload).to.exist;
+      expect(newUpload.bucket).to.be.equal(bucket);
+      expect(newUpload.name).to.be.equal(testUpload.name);
+      expect(newUpload.key).to.be.equal(key);
+
+      const newQuota = await QuotaService.getByOwnerID(USER.id);
+      expect(newQuota.used).to.be.equal(oldQuota.used + newUpload.size);
+    });
+  });
+
```

Comment:
Add more tests: 
 - Negative size of update.
 - Update file with size other than 0.
 - Create two updates with same `key, bucket`
 - Create two updates with same `name, parent, owner`

Reply:
done